### DISCUSSION TITLE
Problem with converting

### DISCUSSION THREAD

**Discussion by vasilevskykv** [microsoft/onnxruntime#14476] [Other Q&A]
Hello. I saved my model to the directory: D:/IDE/VS/source/repos/TensorFlowDL/data/modelLSTM.

When I try to convert it, I write the following:

tf2onnx.convert --saved-model D:/IDE/VS/source/repos/TensorFlowDL/data/modelLSTM --output D:/IDE/VS/source/repos/TensorFlowDL/data/modelLSTM.onnx --opset 13 

or

tf2onnx.convert --saved-model D:\IDE\VS\source\repos\TensorFlowDL\data\modelLSTM --output D:\IDE\VS\source\repos\TensorFlowDL\data\modelLSTM.onnx --opset 13 --

I get a syntax error: "File "<stdin>", line 1
    tf2onnx.convert --saved-model D:\IDE\VS\source\repos\TensorFlowDL\data\modelLSTM --output D:\IDE\VS\source\repos\TensorFlowDL\data\modelLSTM.onnx --opset 13
                                  ^
SyntaxError: invalid syntax"

I cant't understand where is the error

**Comment by ytaous** [comment]
have u tried cmd like so?
python -m tf2onnx.convert --saved-model "saved_model1" --output "model1.onnx"

**Comment by vasilevskykv** [comment]
python -m tf2onnx.convert --saved-model "ModelToConvert ''--output "ModelToConvert.onnx"
Here is the result:
`C:\Users\Laptop\AppData\Local\Programs\Python\Python310\lib\runpy.py:126: RuntimeWarning: 'tf2onnx.convert' found in sys.modules after import of package 'tf2onnx', but prior to execution of 'tf2onnx.convert'; this may result in unpredictable behaviour warn(RuntimeWarning(msg)) usage: convert.py [-h] [--input INPUT] [--graphdef GRAPHDEF] [--saved-model SAVED_MODEL] [--tag TAG] [--signature_def SIGNATURE_DEF] [--concrete_function CONCRETE_FUNCTION] [--checkpoint CHECKPOINT] [--keras KERAS] [--tflite TFLITE] [--tfjs TFJS] [--large_model] [--output OUTPUT] [--inputs INPUTS] [--outputs OUTPUTS] [--ignore_default IGNORE_DEFAULT] [--use_default USE_DEFAULT] [--rename-inputs RENAME_INPUTS] [--rename-outputs RENAME_OUTPUTS] [--use-graph-names] [--opset OPSET] [--dequantize] [--custom-ops CUSTOM_OPS] [--extra_opset EXTRA_OPSET] [--load_op_libraries LOAD_OP_LIBRARIES] [--target {rs4,rs5,rs6,caffe2,tensorrt,nhwc}] [--continue_on_error] [--verbose] [--debug] [--output_frozen_graph OUTPUT_FROZEN_GRAPH] [--inputs-as-nchw INPUTS_AS_NCHW] [--outputs-as-nchw OUTPUTS_AS_NCHW] convert.py: error: unrecognized arguments: ModelToConvert.onnx` 

And finally the file "ModelToConvert.onnx" isn't created.

**Reply by ytaous** [reply]
is this right? "ModelToConvert ''--output  => "ModelToConvert'' --output

**Comment by vasilevskykv** [comment]
Yes, it is right

**Comment by vasilevskykv** [comment]
I made some changes:
python -m tf2onnx.convert --saved-model "ModelToConvert ''--output "MyModel.onnx"
File MyModel.onnx is being created, but then I get the following:
`Exception thrown:
Unable to handle object of type <class 'builtin_function_or_method'>`

**Reply by ytaous** [reply]
I mean t ''--o  there should be space after ", and no space before ". Just want to confirm.

**Comment by vasilevskykv** [comment]
Everything with correspondent spaces is in my line

**Reply by ytaous** [reply]
> Everything with correspondent spaces is in my line

To be clear: this cmd is not right. 
python -m tf2onnx.convert --saved-model "ModelToConvert ''--output "MyModel.onnx"

Should be:
python -m tf2onnx.convert --saved-model "ModelToConvert'' --output "MyModel.onnx"

**Comment by ytaous** [comment]
> Hello. I saved my model to the directory: D:/IDE/VS/source/repos/TensorFlowDL/data/modelLSTM.
> 
> When I try to convert it, I write the following:
> 
> tf2onnx.convert --saved-model D:/IDE/VS/source/repos/TensorFlowDL/data/modelLSTM --output D:/IDE/VS/source/repos/TensorFlowDL/data/modelLSTM.onnx --opset 13
> 
> or
> 
> tf2onnx.convert --saved-model D:\IDE\VS\source\repos\TensorFlowDL\data\modelLSTM --output D:\IDE\VS\source\repos\TensorFlowDL\data\modelLSTM.onnx --opset 13 --
> 
> I get a syntax error: "File "", line 1 tf2onnx.convert --saved-model D:\IDE\VS\source\repos\TensorFlowDL\data\modelLSTM --output D:\IDE\VS\source\repos\TensorFlowDL\data\modelLSTM.onnx --opset 13 ^ SyntaxError: invalid syntax"
> 
> I cant't understand where is the error

can u pls try to double quoted the path, or test it locally wo/ path and see?
--saved-model "modelLSTM" --output "modelLSTM.onnx"
